mbtools/generate_mbz_toc.py

```diff
@@ -37,6 +37,8 @@ def parse_toc(mbz_path):
                     page_dict['activity_name'] = act.name
                     page_dict['lesson_page'] = ""
                     page_dict['visible'] = act.is_visible()
+                    page_dict['lesson_page_type'] = 'content'
```

Comment:
Let's leave this empty for page activities like we do for `lesson_page`. The `lesson_page_type` should only get a value if `lesson_page` does.